### PR TITLE
chore(deps): setup renovate and bump images for examples

### PR DESCRIPTION
**Pull request by phisco**
### Description of your changes

Setup renovate to use https://github.com/renovatebot/renovate/pull/25896, also bumps the used images to the latest available.

I have:

- [x] Read and followed Crossplane's [contribution process].
- [ ] ~Added or updated unit tests for my change.~

[contribution process]: https://git.io/fj2m9
[docs]: https://docs.crossplane.io/contribute/contribute

### CODE REVIEW

File: example/functions/fromYaml/functions.yaml
Comment (HonkingGoose) [inline example/functions/fromYaml/functions.yaml:6]: Looks like there are some end of file problems?

Comment (phisco) [reply example/functions/fromYaml/functions.yaml]: fixed 👍

**Comment by HonkingGoose** [comment]
By the way your Renovate config file still uses the `config:base` preset, you should change it to `config:recommended`, because Renovate `v36` changed the term. 😉 That way you can always search for the config name and find the correct docs.

If you're using the Mend hosted app version of Renovate, you may want to wait a while before merging your PR. It takes a while before the hosted app gets updated to a newer version of Renovate!

- [Renovate docs for `config:recommended` preset](https://docs.renovatebot.com/presets-config/#configrecommended)
- [Renovate docs, when is the Mend Renovate app updated?](https://docs.renovatebot.com/faq/#when-is-the-mend-renovate-app-updated-with-new-renovate-versions)

**Comment by phisco** [comment]
@HonkingGoose, we do use the version hosted by Mend, any pointer to check the version being used?

**Comment by HonkingGoose** [comment]
I think the only way is going to the Mend dashboard (See link in a Renovate PR footer) and reading the debug logs for a recent Renovate run. The debug logs always have a line that says what Renovate version ran.

Please subscribe to this discussion I opened to ask a Renovate maintainer:

- https://github.com/renovatebot/renovate/discussions/25913

**Review by ezgidemirel — APPROVED** [review]
Thanks @phisco

**Pull request merged**
